### DISCUSSION TITLE
User ID in langfuse is missing as of now

### DISCUSSION THREAD

**Discussion by qymab** [BerriAI/litellm#3539] [Q&A]
Hi,

I used to receive the User ID from litellm in the Langfuse dashboard, but now it is missing. However, the Trace ID is still there. What are the recent changes to the parameters that would include the User ID as part of the litellm completion?

Thanks

**Comment by Manouchehri** [comment]
Converted to an issue, tracked here: https://github.com/BerriAI/litellm/issues/3560

**Comment by ghost** [comment]
can't think of any of the top of my head - but i can look into this + add a test to make sure there's no future regressions @qymab

**Reply by qymab** [reply]
Thanks, it's working now.
